app/javascript/components/enrollment/steps/ExposureInformation.js

```diff
@@ -420,16 +420,32 @@ class ExposureInformation extends React.Component {
               {this.state.errors['potential_exposure_country']}
             </Form.Control.Feedback>
           </Form.Group>
-          <Form.Group as={Col} lg={{ span: 8, order: 4 }} md={{ span: 12, order: 3 }} xs={{ span: 24, order: 2 }} className="pl-1">
-            <Form.Check
-              size="lg"
-              label={`CONTINUOUS EXPOSURE${schema?.fields?.continuous_exposure?._whitelist?.list?.has(true) ? ' *' : ''}`}
-              id="continuous_exposure"
-              className="ml-1 d-inline"
-              checked={!!this.state.current.patient.continuous_exposure}
-              onChange={this.handleChange}
-            />
-            <InfoTooltip tooltipTextKey="continuousExposure" location="right"></InfoTooltip>
+          <Form.Group
+            as={Col}
+            lg={{ span: 8, order: 4 }}
+            md={{ span: 12, order: 3 }}
+            xs={{ span: 24, order: 2 }}
+            controlId="continuous_exposure"
+            className="pl-1">
+            <span data-for="disabled-continuous-exposure" data-tip="">
+              <Form.Check
+                size="lg"
+                className="ml-1 d-inline"
+                checked={!!this.state.current.patient.continuous_exposure}
+                onChange={this.handleChange}
+                disabled={this.state.current.patient.isolation}
```

Comment:
Identified the following edge case to address. Tim is aware as I DMed him about it. The edge case is as follows:

Let’s say I start to enroll a case into the Isolation Dashboard and on the initial Identification page, I change the workflow from Isolation to Exposure in the top drop down. When I get to the to the Exposure Info screen, I am not able to select Continuous Exposure (even though I changed the enrollment to be into the Exposure workflow). And similarly, if I start to enroll a monitoree into the Exposure Dashboard, but then change the workflow in the Identification drop down to be Isolation, I can enable Continuous Exposure if I navigate to the Exposure Info screen.

Reply:
this should be as simple as changing `this.state.current.patient.isolation` to `this.props.currentState.isolation` for the disable and show tooltip flags

Reply:
thanks @sgober and @hackrm https://github.com/SaraAlert/SaraAlert/pull/1241/commits/767a37b39ca0d2050eefe2739549b4971f5a69b7

there's also a possibility that a patient is enrolled in exposure and CE is toggled on and then once the enroller gets to the review page, they can edit the workflow to isolation and still end up with a patient in isolation with CE, is there anything we want to do about that edge case?

Reply:
This should fix the issue: https://github.com/SaraAlert/SaraAlert/pull/1241/commits/a9bfd62b04b4765249d51dbe850eae4f142e107e

I don't think this screws anything else with the enrollment state management but please please test!

Reply:
tested it and it works for me! thanks @sgober

Reply:
Seems all good now!